### ISSUE TITLE
Using devtools service for performance tests with browser.reloadSession() throws error when browser.url is called and hangs the test runner

### ISSUE THREAD

**Issue by pmerwin** [webdriverio/webdriverio#4635]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.15.0
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async: sync
 - **Node.js version:** 8.10
 - **NPM version:** 6.11.2
 - **Browser name and version:** chrome 77
 - **Platform name and version:** OSX
 - **Additional wdio packages used (if applicable):**chromedriver service with chromedriver 77, dev tools service

**Config of WebdriverIO**
```
const chromeArgs = [
    'incognito',
    // 'headless'
];
exports.config = merge(wdioBaseConf.config, {
    maxInstances: 10,
    capabilities: [{
        browserName: 'chrome',
        'goog:chromeOptions': {
            args: chromeArgs,
        }
    }],
    port: 9516,
    path: '/',
    // ...
    services: ['chromedriver', 'devtools'],
    chromeDriverLogs: config.wdio.prefix,
    mochaOpts: {
        ui: 'bdd',
        compilers: ['js:@babel/register'],
        timeout: 9999999,
        bail: false
    },
    reporters: ['spec',
        ['json', {
            outputDir: config.wdio.prefix,
            // eslint-disable-next-line
            outputFileFormat: function () {
                return config.wdio.filename;
            }
        }]
    ]
});
```

**Describe the bug**

We are using browser.reloadSession() if an error occurs when getting performance metrics. When we call browser.url after the session reloads this error is thrown and the session hangs:

```
 (node:21293) UnhandledPromiseRejectionWarning: Error: Protocol error (Network.setCacheDisabled): Session closed. Most likely the page has been closed.
    at CDPSession.send (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/puppeteer-core/lib/Connection.js:180:29)
    at NetworkManager._updateProtocolCacheDisabled (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/puppeteer-core/lib/NetworkManager.js:153:24)
    at NetworkManager.setCacheEnabled (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/puppeteer-core/lib/NetworkManager.js:120:16)
    at Page.setCacheEnabled (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/puppeteer-core/lib/Page.js:849:47)
    at Page.<anonymous> (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/puppeteer-core/lib/helper.js:112:23)
    at DevToolsService._setThrottlingProfile (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/@wdio/devtools-service/build/index.js:171:16)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[0-0] (node:21293) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 6)
```

**To Reproduce**

run this sample test with 

describe('Can load page and get lighthouse scores with 0 throttling', function () {
    before(() => {
        browser.enablePerformanceAudits({
            networkThrottling: 'online',
            cpuThrottling: 0,
            cacheEnabled: true
        });
    });
    it('can load a url, than reload browser session for performance tests', function () {
        browser.url('http://google.com');
        browser.reloadSession();
        browser.url('http://google.com'); // => throws error and hangs
        });
    });

**Expected behavior**

No errors after session reload and browser commands all work, etc

=> 2019-10-15T18:33:22.865Z ERROR @wdio/sync: Error: Protocol error (Tracing.start): Session closed. Most likely the page has been closed.

it looks like devtools is unaware there is a new session iD?

if we change the sample tests to this:
```
   browser.url('http://google.com');
        browser.disablePerformanceAudits()
        browser.reloadSession();
        browser.url('http://google.com');
``` 
the error is not thrown, but we of cousre are not able to do an audit in this state, etc

**Log**
https://gist.github.com/pmerwin/1855f46530c7650d716e6331c9339d41

**Comment by pmerwin** [comment]
I put this together to help explain the issue, hope it helps to clarify:

https://cl.ly/147d81cd8a7d

**Comment by christian-bromann** [comment]
This has been fixed by #5651
